rct229/rulesets/ashrae9012019/section19/section19rule35.py

```diff
@@ -0,0 +1,195 @@
+            "hvac_id_b"
+        ]["zone_list"]:
+            for space_b in find_all(
+                f"$.buildings[*].building_segments[*].zones[?(@.id == {zone_id_b})].spaces[*]",
+                rmi_b,
+            ):
+                if hvac_system_serves_only_labs:
+                    if space_b.get("lighting_space_type") is not None:
+                        are_any_lighting_space_types_defined = True
+                        if (
+                            space_b.get("lighting_space_type")
+                            != LIGHTING_SPACE.LABORATORY_EXCEPT_IN_OR_AS_A_CLASSROOM
+                        ):
+                            hvac_system_serves_only_labs = False
+                    else:
+                        all_lighting_space_types_defined = False
+
+            zone_OA_flow_list_of_schedules_b.append(
+                get_min_oa_cfm_sch_zone(rmi_b, zone_id_b)
+            )
+            zone_OA_flow_list_of_schedules_p.append(
+                get_min_oa_cfm_sch_zone(rmi_p, zone_id_b)
+            )
+
+        aggregated_min_OA_schedule_across_zones_b = (
+            aggregate_min_OA_schedule_across_zones(zone_OA_flow_list_of_schedules_b)
+        )
+        aggregated_min_OA_schedule_across_zones_p = (
+            aggregate_min_OA_schedule_across_zones(zone_OA_flow_list_of_schedules_p)
+        )
+
+        return {
+            "are_any_lighting_space_types_defined": are_any_lighting_space_types_defined,
+            "hvac_system_serves_only_labs": hvac_system_serves_only_labs,
+            "all_lighting_space_types_defined": all_lighting_space_types_defined,
+            "aggregated_min_OA_schedule_across_zones_b": aggregated_min_OA_schedule_across_zones_b,
+            "aggregated_min_OA_schedule_across_zones_p": aggregated_min_OA_schedule_across_zones_p,
+        }
+
+    class HVACRule(PartialRuleDefinition):
+        def __init__(self):
+            super(Section19Rule35.HVACRule, self).__init__(
+                rmrs_used=UserBaselineProposedVals(False, True, False),
+            )
+
+        def get_calc_vals(self, context, data=None):
+            are_any_lighting_space_types_defined = data[
+                "are_any_lighting_space_types_defined"
+            ]
+            hvac_system_serves_only_labs = data["hvac_system_serves_only_labs"]
+            all_lighting_space_types_defined = data["all_lighting_space_types_defined"]
+            aggregated_min_OA_schedule_across_zones_b = data[
+                "aggregated_min_OA_schedule_across_zones_b"
+            ]
+            aggregated_min_OA_schedule_across_zones_p = data[
+                "aggregated_min_OA_schedule_across_zones_p"
+            ]
+
+            modeled_baseline_total_zone_min_OA_flow = sum(
+                aggregated_min_OA_schedule_across_zones_b
+            )
+            modeled_proposed_total_zone_min_OA_flow = sum(
+                aggregated_min_OA_schedule_across_zones_p
+            )
```

Comment:
If we use the values to compute the schedules, in here, we will need to add the unit back.